.github/workflows/update-deps.yaml

```
@@ -0,0 +1,64 @@
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+      - name: Run update command
+        run: |
+          make update/libs
+      - name: Check difference
+        id: check_diff
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:5:31: Double quote to prevent globbing and word splitting

(shellcheck)